app/webpack/observations/uploader/actions/actions.js

```diff
@@ -357,6 +397,10 @@ const actions = class actions {
 
   static movePhoto( photo, toObsCard ) {
     return function ( dispatch ) {
+      const { photos: targetPhotos, sounds: targetSounds } = actions.fileCounts( toObsCard.files );
+      const { photos: movedPhotos, sounds: movedSounds } = actions.fileCounts( [photo.file.file] );
```

Comment:
I think this should be `[photo.file]`. Otherwise I get a bug when I add a sound, duplicate it, then try to drag the sound (not the card) onto the original card.

Reply:
Huh, nice catch. Just out of curiosity, did you find this by click testing different edge cases, or some other means?

Reply:
Just click testing. As I mentioned, we don't have any automated testing of React stuff.

Reply:
All good. I think a lot is just my lack of familiarity with all the cases in these features. I've got nowhere to go but up on that front!